nodebuilder/share/config.go

```diff
@@ -1,42 +1,68 @@
 package share
 
 import (
-	disc "github.com/celestiaorg/celestia-node/share/availability/discovery"
+	"fmt"
+
+	"github.com/celestiaorg/celestia-node/nodebuilder/node"
+	"github.com/celestiaorg/celestia-node/share/availability/discovery"
+	"github.com/celestiaorg/celestia-node/share/availability/light"
 	"github.com/celestiaorg/celestia-node/share/p2p/peers"
 	"github.com/celestiaorg/celestia-node/share/p2p/shrexeds"
 	"github.com/celestiaorg/celestia-node/share/p2p/shrexnd"
 )
 
 type Config struct {
-	// UseShareExchange is a flag toggling the usage of shrex protocols for blocksync.
+	Availability light.Parameters `toml:",omitempty"`
```

Comment:
is `Config` used only for `light` or for both? If for both would it be possible cleaner to extract common part into generic Config and for light use inheritance by embedding generic Config and adding extra field?

Reply:
I think `omitempty` is sufficient for now

Reply:
+1, we don't need to overcomplicate this right now. This will be and should be a part of a larger refactoring